### PR TITLE
language and formatting updates to README.md

### PR DESCRIPTION
**Pull request by larsks**

### CODE REVIEW

**Comment by chkumar246** [comment]
@rbowen @apevec @hguemar Please have a look. It looks perfect from my side.

File: README.md [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
 * Read the issue carefully and go through the links attached to issue.
-* If you aren't able understand, join the IRC channel #rdo on Freenode server.
-* Ping the Mentor regarding the issue, S/He will help you move forward.
-
-## I want to add easyfix, How can I do that?
-* Go to issues and file the issue.
-* The issue should properly define the problem to be solved.
-* Please add tag as project name.
-* If possible add steps how it can be reproduced.
+* If you aren't able understand, join the IRC channel `#rdo` on [Freenode][].
```
Comment (apevec) [inline README.md]: able _to_ understand ?

File: README.md [hunk 2]
```diff
@@ -1,36 +1,43 @@
...
+* Concact the mentor regarding the issue, they will help you move forward.
```
Comment (apevec) [inline README.md]: Contact

File: README.md [hunk 3]
```diff
@@ -1,36 +1,43 @@
...
+## I'm still confused, what should I do?
+* Join the [RDO mailing list](https://www.redhat.com/mailman/listinfo/rdo-list) and introduce yourself, or
+* Join `#rdo` IRC channel on [Freenode][] and say hello.  Someone is generally around and happy to help.
+* The [Packager onboarding guide][onboarding] and [RDO Package Maintainer Guide][maintainer] have useful information.
+
+[onboarding]: https://www.rdoproject.org/documentation/onboarding
+[maintainer]: https://www.rdoproject.org/documentation/rdo-packaging
+
+## I want to become mentor, What should I do?
+* Add your name and irc nick to the [mentor list](#mentors)
```
Comment (apevec) [inline README.md]: s/irc/IRC/

**Review by apevec — CHANGES_REQUESTED** [review]

**Comment by larsks** [comment]
Changed as requested!

**Pull request merged**